iothub/device/src/MethodResponseStatusCode.cs

```diff
@@ -4,22 +4,22 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// Status code for Method Response
+    /// Status code for method response.
     /// </summary>
     public enum MethodResponseStatusCode
```

Comment:
I'd consider removing this class when we go v2. Users don't need our help defining http level status codes.